docs/privacy-policy.md

```diff
@@ -0,0 +1,172 @@
+application program interfaces (APIs), application binary interfaces (ABIs), memory, 
+hardware configuration. All telemetry data includes a universally unique identifier 
+(UUID) that is randomly generated on your computer (the client). The UUID is created
+according to the UUID version 4 specification, see 
+[RFC 4122 section 4.4](https://tools.ietf.org/html/rfc4122#section-4.4). The UUID allows
+Sempervirens to distinguish between users, while not being personally identifiable to
+the Sempervirens project or its downstream consumers.
+
+Your IP address is also gathered as a part of a standard web server log and 
+as part of a private, Sempervirens-only data set. However, your 
+IP address is not included in the usage statistics that are aggregated and shared
+publicly. The associations between IP addresses and UUIDs are stored privately
+on Sempervirens infrastructure and is never shared publically.
+IP addresses and any other personal information are never part of the telemetry data.
+
+Usage statistics are transmitted using standard and open encryption technologies 
+(e.g. TLS) and help us improve future versions of our products are services. 
+
+Once sent to Numfocus, usage statistics are aggregated and made available to the public.
+
+When do we share your information with others?
+----------------------------------------------
+
+When we are fulfilling our mission of being open, we will share releases of non-personal 
+information to make our products better and foster an open source software development 
+community. We will only share non-personal information and try to disclose it in a way 
+that minimizes the risk of you being identified.
+
+We will also share information when the law requires it. We follow the law whenever we 
+receive requests about you from a government or related to a lawsuit. We'll notify you 
+when we're asked to hand over your information in this way, unless we're 
+legally prohibited from doing so. Nothing in  this policy is intended to limit any legal 
+defenses or objections that you may have to a third party's request to disclose your 
+information.
+
+When we believe it is necessary to prevent harm to you or someone else we may release 
+your non-personal information. We will only share your information in this way if we 
+have a good faith belief that it is reasonably necessary to protect the rights, 
+property or safety of you, our other users, Sempervirens, Numfocus or the public.
+
+How do we store and protect your information?
+-------------------------------------------------
+
+We are committed to never storing uniquely identifying personal information. 
+
+We will store your IP address, which is considered non-uniquely identifying personal 
+information. Non-uniquely identifying personal information will never be publicly 
+shared. Access to such data will be granted only to approved Sempervirens maintainers
+on an as-needed basis.
```

Comment:
I think we will have to be much more specific about access to the personal information. This access should be managed by a clearly defined and NumFocus approved set of individuals. Access to PII should be managed by written requests and individuals with access should not ever be allowed to download the data to their own computers. Using a centralized and authorized/authenticated JupyterHub instance is a good approach for managing this access.